FrameworkSystem/scripts/dirac-admin-update-instance.py

```diff
@@ -23,22 +23,28 @@
                        )
 Script.registerSwitch("", "hosts=", "Comma separated list of hosts or file containing row wise list of hosts"
                                     " targeted for update (leave empty for all)")
+Script.registerSwitch("", "retry=", "Number of retry attempts on hosts that have failed to update")
```

Comment:
Don't you want to add default of 1 retry?

Reply:
As the retry is an optional flag, I would opt for having it set to 0. Along the lines "explicit over implicit".